README.md

```diff
@@ -1,18 +1,25 @@
-# CNCF TechDocs office hours
+# CNCF TechDocs
 
-This repository holds team information and meeting notes for tech docs office hours.
+This repository holds resources provided by the CNCF Technical Documentation team. The repo contains the following directories:
+
+- `analysis` contains instructions, templates, and criteria for requesting and performing an analysis of an OSS project's website and technical documentation. Completed analyses are stored here as well.
+- `resources` contains information that OSS teams can use to set up a documentation project as suggested by the Tech Docs team.
```

Comment:
We've traditionally used "TechDocs" 🤷‍♂️, so let's stick to that spelling everywhere, i.e. `s/Tech Docs/TechDocs/g` in all the updates, like this:

```suggestion
- `resources` contains information that OSS teams can use to set up a documentation project as suggested by the TechDocs team.
```

Reply:
I've always thought camel-case spelling was used mostly for brand names -- not something I'd apply to a department name or functional description. I have fairly conservative stylistic taste, though. If you're using it everywhere else then make it universal. Consistency is important as well.